tools/quickstart/quickstart/bridge.py

```diff
@@ -95,3 +112,31 @@ def _show_file_override_dialog(base_dir, bridge_config_file):
             )
         else:
             assert False, "unreachable"
+
+
+def read_json_rpc_url():
+    url = click.prompt("JSON RPC url")
+    if verify_url(url):
+        return url
+    else:
+        url = click.prompt(
+            f"The given url seems invalid {url}, please correct it or enter the same url if valid"
+        )
+        return url
+
+
+def verify_url(url):
```

Comment:
Please write a test for this, as we fucked it up already.

Reply:
Would we rather not use marshmallow then? This way of doing it is what I understood from Ralf's suggestions, but I do not really know why it is better than trusting marshmallow directly for verification.

Reply:
It's a new dependency for the quicktools package. Other than that there's no good reason not to use it.

Reply:
What has writing a test to do with using marshmallow

Reply:
I can understand that we do not trust this verify_url function I wrote and require tests for it. I would trust marshmallow verification and not write tests. But I will just write tests.